tests/lib/rules/space-infix-ops.js

```diff
@@ -24,7 +24,9 @@ eslintTester.addRuleTest("lib/rules/space-infix-ops", {
         "(a + b) + (c + d)",
         "a = b",
         "a ? b : c",
-        "var a = b"
+        "var a = b",
+        { code: "a|0", args: [2, { int32Hint: true }] },
+        { code: "a |0", args: [2, { int32Hint: true }] }
```

Comment:
I think this shouldn't be allowed, the hint should have no spaces before or after the `|`.